command_mode.go

```diff
@@ -34,35 +34,12 @@ func handleCommandModeArgs(arguments map[string]interface{}) {
 }
 
 func handleBluePrintMode(arguments map[string]interface{}) {
-	doCreate := arguments["--create-api"]
+	doCreate := arguments["--create-api"].(bool)
```

Comment:
This will panic if it's not a bool type, e.g. if it's actually just nil. Are you 100% certain that it will always be of bool type? Have you tried to get it to crash by using weird parameters like `--create-api=foo`?